src/test/java/api/exercise/Exercise1.java

```diff
@@ -0,0 +1,110 @@
+package api.exercise;
+
+import java.util.function.Function;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused", "MismatchedQueryAndUpdateOfCollection"})
+public class Exercise1 {
+
+    enum Status {
+        UNKNOWN,
+        PENDING,
+        COMMENTED,
+        ACCEPTED,
+        DECLINED
+    }
+
+    @Test
+    public void acceptGreaterThan21OthersDecline() {
+        Person alex = new Person("Алексей", "Мельников", 20);
+        Person ivan = new Person("Иван", "Стрельцов", 24);
+        Person helen = new Person("Елена", "Рощина", 22);
+        Map<Person, Status> candidates = new HashMap<>();
+        candidates.put(alex, Status.PENDING);
+        candidates.put(ivan, Status.PENDING);
+        candidates.put(helen, Status.PENDING);
+
+        // Реализация
+        candidates.replaceAll(
+                (person, status) -> person.getAge() >= 21 ? Status.ACCEPTED : Status.DECLINED);
```

Comment:
I think, "greater than 21" means that we shouldn't include 21.

Reply:
I agree with you, didn't think about it